provider/bikeshed/data/debug/units/essentials@1/narrow/en.json

```diff
@@ -1,5 +1,197 @@
 {
   "prefixes": [
+    [
```

Comment:
observation: the SI prefixes have a lot of deduplication potential, which is not being realised because the `per` and `times` strings are very locale dependent.

Reply:
I was thinking in a next PR to have a fall_back to the en_us to remove redundancies.

Reply:
that doesn't generally work, because users might build data without `en-US`